mvc/src/main/java/com/interface21/webmvc/servlet/mvc/InvocableHandlerMethod.java

```diff
@@ -0,0 +1,39 @@
+package com.interface21.webmvc.servlet.mvc;
+
+import com.interface21.webmvc.servlet.MethodArgumentResolutionException;
+import com.interface21.webmvc.servlet.mvc.model.HandlerMethod;
+import com.interface21.webmvc.servlet.mvc.model.MethodParameter;
+import com.interface21.webmvc.servlet.mvc.resolver.HandlerMethodArgumentResolverComposite;
+import jakarta.servlet.http.HttpServletRequest;
+import java.lang.reflect.Method;
+import org.apache.commons.lang3.ObjectUtils;
+
+public class InvocableHandlerMethod extends HandlerMethod {
+  private static final Object[] EMPTY_ARGS = new Object[0];
+  private final HandlerMethodArgumentResolverComposite resolvers;
```

Comment:
```suggestion
  private static final Object[] EMPTY_ARGS = new Object[0];
  
  private final HandlerMethodArgumentResolverComposite resolvers;
```
상수와 상태값을 구분해주면 좋습니다 :)
>https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace